Example/ViewController.swift

```diff
@@ -725,6 +725,25 @@ extension ViewController: NavigationMapViewDelegate {
         guard let index = routes?.firstIndex(where: { $0 === route }) else { return }
         currentRouteIndex = index
     }
+    
+    func navigationMapView(_ navigationMapView: NavigationMapView, didTap routeWithDuration: Route) {
```

Comment:
I think it would be confusing to have both `navigationMapView(_:didSelect:)` and `navigationMapView(_:didTap:)`: since tapping selects a route, a developer might accidentally implement only one method without realizing that the other also needs to be implemented. Is there a need to specifically distinguish between tapping on the route line versus tapping on a route duration? If not, let’s just call `navigationMapView(_:didSelect:)` in either case.

Reply:
Thanks for the suggestion, the `navigationMapView(_:didTap:)` is moved. When tapping on the route duration bubble, the `navigationMapView(_:didSelect:)` will be called.